docker-storage-setup.sh

```diff
@@ -435,11 +438,11 @@ setup_lvm_thin_pool () {
   tpool=`get_configured_thin_pool`
 
   if [ -n "$tpool" ]; then
-     local escaped_pool_lv_name=`echo $POOL_LV_NAME | sed 's/-/--/g'`
+     local escaped_pool_lv_name=`echo $CONTAINER_THINPOOL | sed 's/-/--/g'`
```

Comment:
did you get rid of checks which did not allow passing in only inputfile. If user specifies only inputfile and not outputfile, it should error out.